api/server/controllers/optionController.js

```diff
@@ -0,0 +1,67 @@
+Option = require('../models/optionModel')
+
+exports.create = function(req, res) {
+    let option = new Option();
+    option.time = req.params.time;
+    option.votes = req.params.votes;
+    option.label = req.params.label;
+    
+    option.dispatchEvent(function(err) {
+        if(err) {
+            res.json({
+                status: 500,
+                erroMessage: err.message,
+                errorName: err.name,
+            })
+        }
+        res.json({
+            status: res.statusCode,
+            message: "Option created successfully",
+            option: option,
+        })
+    })
+}
+
+// Update option label
+exports.updateLabel = function(req, res) {
```

Comment:
You can delete labels from Option's, I think we agreed on this earlier? Correct me if I'm wrong though lol.